test/Microsoft.ML.TestFramework/BaseTestPredictorsMaml.cs

```diff
@@ -354,7 +354,7 @@ public string GetLoaderTransformSettings(TestDataset dataset)
             string[] extraSettings = null, string extraTag = "", bool summary = false, int digitsOfPrecision = DigitsOfPrecision)
         {
             Contracts.Assert(IsActive);
-            Run_TrainTest(predictor, dataset, extraSettings, extraTag, summary: summary, digitsOfPrecision: digitsOfPrecision);
+           // Run_TrainTest(predictor, dataset, extraSettings, extraTag, summary: summary, digitsOfPrecision: digitsOfPrecision);
```

Comment:
Was this a mistake?

Reply:
thanks for this, commented out while debugging. 

---
In reply to: [224863064](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224863064) [](ancestors = 224863064)